include/seqan3/alignment/matrix/edit_distance_score_matrix_full.hpp

```diff
@@ -39,6 +39,9 @@ class edit_distance_score_matrix_full
               typename edit_traits>
     friend class edit_distance_unbanded;
 
+    template <typename, typename>
```

Comment:
please add a comment for why this needs to befriended. That might be obvious for you as the dev but other people might sumble over it when they have to maintain it.

Reply:
It would also be good to name the types also it does not change the behavior but it makes it easier to understand of what is goning on or at least expected